### PR TITLE
#265 fix mobile display for menus and features page

### LINKED ISSUE #265: [Enhancement] Fix mobile display for menus and features page

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #265

### CODE REVIEW

File: app/templates/components/elements/x-navigation.hbs [hunk 1]
```diff
@@ -20,17 +22,17 @@
       {{#link-to 'contact'}}Contact{{/link-to}}
     </div>
     {{#if auth.authenticated}}
-      <div class="item">
+      <div class="item butt">
         {{#link-to 'account' class="ui outlined blue button"}}Go To Dashboard{{/link-to}}
       </div>
     {{else}}
-      <div class="item">
+      <div class="item butt">
```
Comment (James1x0) [inline app/templates/components/elements/x-navigation.hbs]: ?

File: app/styles/granite/components/elements/nav.less [hunk 2]
```diff
@@ -98,3 +98,107 @@
     }
   }
 }
+
+
+@media screen and (min-width: @screen-md1) and (max-width: 767px) {
```
Comment (James1x0) [inline app/styles/granite/components/elements/nav.less]: Indentation needs help here.

File: app/templates/components/display/product/features-list.hbs [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (James1x0) [inline app/templates/components/display/product/features-list.hbs]: ✂️

File: app/templates/components/display/product/features-list.hbs [hunk 4]
```diff
@@ -6,6 +7,9 @@
   {{/each}}
 </ul>
 
+
+
+
```
Comment (James1x0) [inline app/templates/components/display/product/features-list.hbs]: ✂️

File: app/templates/components/elements/x-navigation.hbs [hunk 5]
```diff
@@ -6,7 +6,9 @@
 </div>
 
 {{#if topLevel}}
-  <div class="right menu">
+
+
```
Comment (James1x0) [inline app/templates/components/elements/x-navigation.hbs]: ✂️

File: app/templates/components/elements/x-navigation.hbs [hunk 6]
```diff
@@ -6,7 +6,9 @@
...
+<div class="right menu ">
```
Comment (James1x0) [inline app/templates/components/elements/x-navigation.hbs]: "right menu " => "right menu"

File: app/templates/components/elements/x-navigation.hbs [hunk 7]
```diff
@@ -20,17 +22,17 @@
...
+      <div class="item login_or_dashboard">
         <div class="ui buttons">
           {{#link-to 'login' class="ui blue button"}}Log in{{/link-to}}
           <div class="or"></div>
           {{#link-to 'signup' class="ui green button"}}Sign Up{{/link-to}}
         </div>
       </div>
     {{/if}}
-  </div>
+</div>
```
Comment (James1x0) [inline app/templates/components/elements/x-navigation.hbs]: Indentation issue

File: app/templates/our-product.hbs [hunk 8]
```diff
@@ -1,6 +1,8 @@
+<div class="our-product">
```
Comment (James1x0) [inline app/templates/our-product.hbs]: Our-product block contents need to be indented.

Comment (JavaLavaMT) [reply app/templates/our-product.hbs]: thank goodness for shift+command+ L

Comment (JavaLavaMT) [reply app/templates/our-product.hbs]: not sure what you mean here?

**Pull request merged**